package/test/beamer_delegate_test.dart

```diff
@@ -402,13 +402,19 @@ void main() {
       rootDelegate.beamToNamed('/test'); // initial will update
       await tester.pump();
       expect(rootDelegate.configuration.location, '/test');
-      expect(childDelegate.configuration.location, '/test');
+      childDelegate.updateFromParent
+          ? expect(childDelegate.configuration.location, '/test')
+          : expect(
+              childDelegate.configuration.location, childDelegate.initialPath);
```

Comment:
As this is the test that specifically declares `updateFromParent: false`, we should have just
```dart
expect(childDelegate.configuration.location, childDelegate.initialPath);
```